task/grype/0.1/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Although if this is standard across all pre-apply-task-hook.sh files just leave it

Reply:
ok